librad/Cargo.toml

```diff
@@ -47,6 +47,7 @@ tempfile = "3.1"
 thiserror = "1.0"
 time = "0.2"
 tracing = "0.1"
+tracing-attributes = "<0.12.0, ^0.1.13"
```

Comment:
Why the bound to be smaller than `0.12.0`?

Reply:
It allows dependents to pick a `tracing-attributes` lower than `0.12.0` so that they're not pinned to the `0.1.13` or above :)

Reply:
cargoooooooooooooooooo

Reply:
:car::dash: